src/neptune/common/hardware/gpu/gpu_monitor.py

```diff
@@ -62,14 +62,14 @@ def __nvml_get_or_else(self, getter, default=None):
         try:
             nvmlInit()
             return getter()
-        except NVMLError as e:
+        except NVMLError:
             if not GPUMonitor.nvml_error_printed:
                 warning = (
                     "Info (NVML): %s. GPU usage metrics may not be reported. For more information, "
                     "see https://docs-legacy.neptune.ai/logging-and-managing-experiment-results"
                     "/logging-experiment"
                     "-data.html#hardware-consumption "
                 )
```

Comment:
Under which circumstances does this show? Just wondering whether we should update the ancient docs link.

Reply:
Yep! And if the page is not up to snuff, I'll fix it 😉